pkg/test/resourcefixture/testdata/basic/bigtable/v1beta1/bigtableappprofile/_generated_object_bigtableappprofile.golden.yaml

```diff
@@ -22,7 +21,6 @@ spec:
   - cluster1-${uniqueId}
   - cluster2-${uniqueId}
   multiClusterRoutingUseAny: true
-  resourceID: bigtableappprofile-${uniqueId}
```

Comment:
@maqiuyujoyce For TF-based controller, we are writing "resourceID" back to Spec even though we set "state-into-spec: absent"?

Reply:
Yes, we treat `spec.resourceID` differently in TF-based/DCL-based controllers, and we always write them back. E.g. https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/pkg/krmtotf/tftokrm.go#L229